### ISSUE TITLE
Improve whitespace formatting in `pulumi convert` output

### ISSUE THREAD

**Issue by jkodroff** [pulumi/pulumi#15211]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

When running `pulumi convert`, the output doesn't have much in the way of whitespace:

<img width="1161" alt="image" src="https://github.com/pulumi/pulumi/assets/410614/0768983f-029c-4ad9-9a77-56861760c8e7">

Could we put a blank line in between resource/function declarations for better readability?

**Comment by Frassle** [comment]
I don't think this is due to convert per-se but just how nodejs does any program generation.

**Comment by jkodroff** [comment]
This isn't relevant to me anymore. Closing.
